<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>679</width>
    <height>434</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>679</width>
    <height>434</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>679</width>
    <height>434</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Mona Sans Light</family>
    <pointsize>16</pointsize>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>AquaSense</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>679</width>
      <height>434</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>679</width>
      <height>434</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>679</width>
      <height>434</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Mona-Sans Black Wide</family>
      <pointsize>30</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="currentIndex">
     <number>4</number>
    </property>
    <widget class="QWidget" name="welcome_page">
     <widget class="QLabel" name="label_for_logo">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>79</y>
        <width>300</width>
        <height>301</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="assets.qrc">:/icons/assets/icon-1024x1024.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_for_name">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>679</width>
        <height>71</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>679</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>679</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Mona-Sans Black Wide</family>
        <pointsize>30</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>AquaSense</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_for_version">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>360</y>
        <width>679</width>
        <height>31</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>679</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>679</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Mona-Sans Medium Narrow</family>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>v0.0.1</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="back_to_main">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>400</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Mona-Sans Medium Wide</family>
        <pointsize>9</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Next</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="main_page">
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>20</y>
        <width>672</width>
        <height>130</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontal_layout_main">
       <item>
        <widget class="QLabel" name="label_for_logo_main">
         <property name="minimumSize">
          <size>
           <width>128</width>
           <height>128</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>128</width>
           <height>128</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="assets.qrc">:/icons/assets/icon-200x200.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_for_intro_main">
         <property name="font">
          <font>
           <family>Mona-Sans Black Wide</family>
           <pointsize>25</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Simplicity in Water Wisdom</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="back_to_welcome">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>400</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Mona-Sans Medium Wide</family>
        <pointsize>9</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="group_box_main">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>199</y>
        <width>679</width>
        <height>181</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Mona-Sans Medium Wide</family>
        <pointsize>9</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>0</y>
         <width>291</width>
         <height>181</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="back_to_data">
          <property name="font">
           <font>
            <family>Mona-Sans Medium Wide</family>
            <pointsize>9</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Entries</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="back_to_insights">
          <property name="text">
           <string>Insights</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="back_to_visualizations">
          <property name="text">
           <string>Visualizations</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="back_to_contact_main">
          <property name="font">
           <font>
            <family>Mona-Sans Medium Wide</family>
            <pointsize>9</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Contact the developers</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="contact_page">
     <widget class="QLabel" name="label_for_contact">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>30</y>
        <width>679</width>
        <height>41</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>679</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Mona-Sans Black Wide</family>
        <pointsize>30</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Contact the developers</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="Line" name="line_for_contact">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>70</y>
        <width>679</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>679</width>
        <height>0</height>
       </size>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="back_to_main_contact">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>380</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Mona-Sans Medium Wide</family>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>160</y>
        <width>571</width>
        <height>151</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="vertical_layout_contact"/>
     </widget>
    </widget>
    <widget class="QWidget" name="data_page">
     <widget class="QLabel" name="label_for_data">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>30</y>
        <width>679</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Manage your water samples</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="Line" name="line_for_data">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>70</y>
        <width>679</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QTableWidget" name="table_for_data">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>661</width>
        <height>291</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Mona Sans Medium</family>
        <pointsize>10</pointsize>
        <bold>false</bold>
       </font>
      </property>
     </widget>
     <widget class="QPushButton" name="back_to_main_data">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>398</y>
        <width>80</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Mona-Sans Medium Wide</family>
        <pointsize>8</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="insights_page">
     <widget class="QPushButton" name="back_to_main_insights">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>400</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Mona-Sans Medium Wide</family>
        <pointsize>9</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_for_insights">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>30</y>
        <width>679</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>See through your water</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>70</y>
        <width>679</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>99</y>
        <width>661</width>
        <height>151</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Mona-Sans Medium Wide</family>
        <pointsize>9</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QWidget" name="formLayoutWidget_3">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>9</y>
         <width>271</width>
         <height>111</height>
        </rect>
       </property>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="pHLabel">
          <property name="text">
           <string>pH</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="phLineEdit"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="hardnessLabel">
          <property name="text">
           <string>Hardness</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="hardnessLineEdit"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="turbidityLabel">
          <property name="text">
           <string>Turbidity</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="turbidityLineEdit"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="tDSLabel">
          <property name="text">
           <string>TDS</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="solidsLineEdit"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="formLayoutWidget_4">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>10</y>
         <width>251</width>
         <height>136</height>
        </rect>
       </property>
       <layout class="QFormLayout" name="formLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="chloraminesLabel">
          <property name="text">
           <string>Chloramines</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="chloraminesLineEdit"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="sulfateLabel">
          <property name="text">
           <string>Sulfate</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="sulfateLineEdit"/>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="organicCarbonLineEdit"/>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="conductivityLineEdit"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="organicCarbonLabel">
          <property name="text">
           <string>Organic Carbon</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="conductivityLabel">
          <property name="text">
           <string>Conductivity</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="trihalomethanesLineEdit"/>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="trihalomethanesLabel">
          <property name="text">
           <string>Trihalomethanes</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget_3">
       <property name="geometry">
        <rect>
         <x>570</x>
         <y>11</y>
         <width>82</width>
         <height>101</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="vertical_layout_insights">
        <item>
         <widget class="QPushButton" name="predict_button">
          <property name="font">
           <font>
            <family>Mona-Sans Medium Wide</family>
            <pointsize>8</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Predict</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="reset_entries_button">
          <property name="font">
           <font>
            <family>Mona-Sans Medium Wide</family>
            <pointsize>8</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Reset</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="Line" name="line_2">
       <property name="geometry">
        <rect>
         <x>17</x>
         <y>130</y>
         <width>261</width>
         <height>20</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="predictionLabel">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>270</y>
        <width>679</width>
        <height>141</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Mona-Sans Black Wide</family>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="visualization_page">
     <widget class="QLabel" name="label_for_visualization">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>30</y>
        <width>679</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Visualizations</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="Line" name="line_for_visualization">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>70</y>
        <width>679</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="back_to_main_visualization">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>400</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Mona-Sans Medium Wide</family>
        <pointsize>9</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_for_graphs">
      <property name="geometry">
       <rect>
        <x>58</x>
        <y>110</y>
        <width>561</width>
        <height>221</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="previous_button_visualization">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>360</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Mona-Sans Medium Wide</family>
        <pointsize>9</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Previous</string>
      </property>
     </widget>
     <widget class="QPushButton" name="next_button_visualization">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>360</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Mona-Sans Medium Wide</family>
        <pointsize>9</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Next</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="assets.qrc"/>
 </resources>
 <connections/>
</ui>
